.github/workflows/lint.yml

```diff
@@ -0,0 +1,21 @@
+name: lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: technote-space/get-diff-action@v1
+      id: git_diff
+      with:
+        SUFFIX_FILTER: .clj
+        SEPARATOR: ' '
+    - uses: xcoo/clj-lint-action@v1.0.0
+      with:
+        linters: "[\"clj-kondo\" \"kibit\" \"eastwood\"]"
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        runner: ":leiningen"
+        usefiles: "true"
+        files:  ${{ steps.git_diff.outputs.diff }}
```

Comment:
Can this be done without using `technote-space/get-diff-action`?
I want to have fewer dependencies on unofficial GitHub actions.